envoy-control-runner/build.gradle

```diff
@@ -34,3 +35,7 @@ test {
     useJUnitPlatform()
 }
 
+run {
+    systemProperties.putIfAbsent("spring.profiles.active", "local")
```

Comment:
why do we need that?

Reply:
Because right now EC doesn't work out-of-the-box as described in docs quickstart section: https://envoy-control.readthedocs.io/en/latest/quickstart/

If you run ./gradlew run, "local" profile is not active and application-local.yaml is not read. So we don't have proper consul port and EC fails to start.

This is fix for that.